sw/device/tests/power_virus_systemtest.c

```diff
@@ -108,9 +120,24 @@ enum {
   kI2c1TargetAddress = 0x02,
   kI2c2TargetAddress = 0x03,
   /**
-   * UART parameters
+   * UART parameters.
    */
   kUartFifoDepth = 32,
+  /**
+   * Pattern Generator parameters.
+   */
+  kPattgenClockDivisor = 0,
```

Comment:
Oof, our API makes for some awkward propagation of the hardware's encoding alongside a deceptive name, doesn't it? We're not _really_ going to be dividing by 0, heh.

Reply:
oh yeah, I can update the API in a follow up.